packages/financial-templates-lib/test/price-feed/CreatePriceFeed.js

```diff
@@ -331,20 +335,20 @@ contract("CreatePriceFeed.js", function (accounts) {
       withdrawalLiveness: "1000",
       collateralAddress: collateralTokenAddress,
       tokenAddress: syntheticTokenAddress,
-      finderAddress: finder.address,
+      finderAddress: finder.options.address,
       priceFeedIdentifier: padRight(utf8ToHex("ETH/BTC"), 64), // Note: an identifier which is part of the default config is required for this test.
       liquidationLiveness: "1000",
       collateralRequirement: { rawValue: toWei("1.5") },
-      disputeBondPct: { rawValue: toWei("0.1") },
```

Comment:
why was this change required? does this imply you are now using the new version of these contracts from core/contracts-node? I might have missed that change in the `package.json`

Reply:
Yeah, we're using the new version of the EMP. I _think_ this is better because the new EMP is all we're supporting these days, so it seems better to test against it as opposed to the older, hardcoded versions that don't exist in the wild anymore.

Reply:
agreed! all the old ones are deprecated anyways